apps/etterlatte-behandling/src/main/kotlin/behandling/aktivitetsplikt/AktivitetspliktService.kt

```diff
@@ -40,6 +44,28 @@ class AktivitetspliktService(
         return hentAktivitetspliktOppfolging(behandlingId)!!
     }
 
+    fun oppfyllerAktivitetsplikt(
```

Comment:
Denne er nok litt for "dum" i lengden når vi etterhvert støtter at saksbehandler skal kunne legge inn aktivitetsgrad/unntak utover oppgaven på 4 mnd, men for akkurat nå så tror jeg logikken er god nok.